region/maridia/inner-green/Shaktool Room.json

```diff
@@ -82,6 +87,31 @@
     }
   ],
   "strats": [
+    {
+      "link": [1, 1],
+      "name": "R-Mode Entry",
+      "entranceCondition": {
+        "comeInWithRMode": {}
+      },
+      "requires": [],
+      "flashSuitChecked": true,
+      "blueSuitChecked": true,
+      "clearsObstacles": ["R-Mode"]
+    },
```

Comment:
```suggestion
```

Reply:
This entry would allow collecting the digging flag and getting bluesuit in a single room entry.

As an example for MapRando, it would allow collecting the `f_ShaktoolDoneDigging` flag in a situation where a bluesuit is needed to return from doing so. Without this, the return path can't re-enter to get blue suit (because it'd be trying to use the digging flag when it hasn't confirmed it is reachable yet), another way to return would have to open up before the flag could be collected and then the room used for bluesuits.